src/Microsoft.OData.Core/DependsOnIdsTracker.cs

```diff
@@ -0,0 +1,90 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="DependsOnIdsTracker.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData
+{

```

Comment:
If this is only used by multipart/mixed, should it go in the multipart/mixed namespace?

Reply:
yes, moved.

---
In reply to: [160557298](https://github.com/OData/odata.net/pull/1020#discussion_r160557298) [](ancestors = 160557298)